IQSS/dataverse
# better message when picking a new password #9326

**What this PR does / why we need it**:

Users get a confusing message when resetting their password. This pull request provides a better message.

**Which issue(s) this PR closes**:

- Closes #9326

**Special notes for your reviewer**:

The only place `passwdReset.newPasswd.details` is used is in `passwordreset.xhtml` (via `PasswordResetPage.customPasswordResetAlertMessage()`.

I found the wording ("Please pick a strong password that matches the criteria below.") in the git history.

**Suggestions on how to test this**:

Option 1:

- Create a new user via API with a password that is non-compliant, such as "admin"
- Log in as the new user and see the prompt

Option 2:

- Go through the password reset process.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

Yes, here is the new message in the existing blue alert box, in context:

![Screen Shot 2023-06-16 at 2 51 30 PM](https://github.com/IQSS/dataverse/assets/21006/e99db8e0-6874-4e79-8f14-098f0638ee3d)

**Is there a release notes update needed for this change?**:

No, it's a minor change.

**Additional documentation**:

None.
